sql/core/src/main/scala/org/apache/spark/sql/jdbc/MySQLDialect.scala

```diff
@@ -52,6 +52,15 @@ private case class MySQLDialect() extends JdbcDialect with SQLConfHelper with No
     supportedFunctions.contains(funcName)
 
   class MySQLSQLBuilder extends JDBCSQLBuilder {
+    override def visitExtract(field: String, source: String): String = {
+      field match {
+        case "DAY_OF_YEAR" => s"DAYOFYEAR($source)"
+//        case "DAY_OF_WEEK" => s"(EXTRACT(ISO_DAY_OF_WEEK FROM $source) % 7) + 1"
+//        case "WEEK_DAY" => s"WEEKDAY($source)"
```

Comment:
Can we remove those commented changes?

Reply:
This PR is WIP now. I will change them later.